### DISCUSSION TITLE
Copilot error on second computer with VSCode

### DISCUSSION THREAD

**Discussion by ahajdar** [community/community#68406] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have an error when after installing Copilot on my second computer running VSCode.  The error is: "Extension activation failed: "Timed out waiting for authentication provider to register"".  Changing DNS didn't help, reinstall didn't help.

I have an individual account and I don't use the extension commercially.

Any ideas what I can do to enable it on second computer as well?

**Comment by Akash1134** [comment]
Hi @ahajdar ,

Sorry about the trouble caused? Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

Thanks!

**Comment by ahajdar** [comment]
Hi, I shouldn't be...is there a way to check from my computer?
